public/index.html

```diff
@@ -0,0 +1,375 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
+  <title>Ballers: Become A Landlord</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png">
+  <link rel="manifest" href="./site.webmanifest">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="./assets/slick/slick.css"/>
```

Comment:
Lets create a folder in `assets/scss` title plugins and include all 3rd party styles for easier maintenance. The css files should be renamed to scss (you don't need to change/update any of the styles, the css styles should not be updated). The 3rd party files should be included in the scss main file (ballers.scss) so that they can be compressed for the main page. 

We are doing this for 2 reasons:
1. It is faster for the browser to load a single file than loading multiple files. Single request loads faster than multiple requests.

2. All SCSS files are compressed which will ensure that the overall file size is smaller.